### PR TITLE
fix: 修复lb问题

### PR DESCRIPTION
**Pull request by dellkeji**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=1687) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=1687) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-bcs?pullRequest=1687) it.</sub>

File: bcs-ui/backend/uniapps/network/views/lb/nodes.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    def _construct_labels_by_op(self, labels: Dict, op: str = LBLabelOp.ADD) -> Dict:
+        if op == LBLabelOp.ADD:
+            labels.update(K8S_LB_LABEL)
+        else:
+            for key in K8S_LB_LABEL:
+                labels[key] = None
+        return labels
+
+
+def convert_ips(access_token: str, project_id: str, cluster_id: str, lb_ips: Dict) -> Dict:
```
Comment (jamesgetx) [inline bcs-ui/backend/uniapps/network/views/lb/nodes.py:58]: 两个Dict的注解可以再细化下

Comment (jamesgetx) [reply bcs-ui/backend/uniapps/network/views/lb/nodes.py:58]: lb_ips像是一个ip列表，考虑换个名字，比如`ip_used_data`之类的

File: bcs-ui/backend/uniapps/network/views/lb/nodes.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    """转换为ip信息
+    NOTE: 历史数据可能为{ip_id: 是否使用}，需要把ip_id转换为ip
+    """
+    nodes = cluster_utils.get_cluster_nodes(access_token, project_id, cluster_id)
+    node_id_ip = {info["id"]: info["inner_ip"] for info in nodes}
+    # 通过ip id, 获取ip
+    _ips = {}
+    for ip, used in lb_ips.items():
+        if not ip.isdigit():
```
Comment (jamesgetx) [inline bcs-ui/backend/uniapps/network/views/lb/nodes.py:67]:
```
try:
    ip_id = int(ip)
    if node_id_ip.get(ip_id):
        _ips[node_id_ip[ip_id]] = used
except ValueError:
    _ips[ip] = used
```

File: bcs-ui/backend/uniapps/network/views/lb/nodes.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import json
+from typing import Dict, List
+
+from backend.container_service.clusters.base import utils as cluster_utils
+from backend.container_service.clusters.base.models import CtxCluster
+from backend.resources.node.client import Node
+from backend.uniapps.network.constants import K8S_LB_LABEL, LBLabelOp
+
+
+class LoadbalancerLabels:
```
Comment (jamesgetx) [inline bcs-ui/backend/uniapps/network/views/lb/nodes.py:24]: 类命名可以调整为`LoadBalancer`或者`LoadBalancerController`之类的。label里面言label，有点像一个Node类里面还有add_node方法这种感觉。

File: bcs-ui/backend/uniapps/network/views/lb/k8s.py [hunk 4]
```diff
@@ -452,8 +363,13 @@ def destroy(self, request, project_id, pk):
         # 标识LB被删除
         self.delete_lb_conf(lb_conf)
         # 删除节点标签
-        delete_node_id_list = [node_id for node_id in json.loads(lb_conf.ip_info)]
-        self.delete_node_label(request, delete_node_id_list, project_id, lb_conf)
+        ip_list = convert_ips(
```
Comment (jamesgetx) [inline bcs-ui/backend/uniapps/network/views/lb/k8s.py:366]: convert_ips返回的应该是Dict[str, bool]的结构？所以涉及到的变量命名需要调整

**Review by jamesgetx — APPROVED** [review]

**Pull request merged**